cdk.schema.json

```diff
@@ -91,6 +91,32 @@
       ],
       "type": "object"
     },
+    "RuleSpecificIntrinsic": {
```

Comment:
There is a [closed list][1] of functions that you can use inside a `Rules` section. We can make the Json schema more specific by only allowing those functions here. I've left it open for now because this is low priority.

[1]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-rules.html#fn-refall